lib/cog/adapters/config.ex

```diff
@@ -1,13 +1,18 @@
 defmodule Cog.Adapters.Config do
+  require Logger
+
   @moduledoc """
-  The Config module is used to ingest adapter configuration that is specified in Mix, apply validations
-  and type coercion, cache it in the state of a GenServer, and make it available to the adapter. To use
-  it, create new module with `@config` and `@schema` attributes defined, and `use Cog.Adapters.Config`.
+  The Config module is used to ingest adapter configuration that is specified
+  in Mix, apply validations and type coercion, cache it in the state of a
+  GenServer, and make it available to the adapter. To use it, create new module
```

Comment:
Is this documentation up to date? This module isn't a genserver anymore. Or am I misunderstanding?